### PR TITLE
Migrate the HUST page to FOSDEM HPC BD DS.

### PR DESCRIPTION
**Pull request by ehiggs**
This will be previewable shortly [here](http://ehiggs.github.io/fosdem2016_hpc_bigdata_devroom).

This need to be pushed to a `gh-pages` branch but PRs in Github don't appear to be able to push to new branches.

TODO: tidy up text for proposal. 
TODO: Create actual introduction text instead of lorum ipsum dorum filler.

### CODE REVIEW

**Comment by JensTimmerman** [comment]
why not create a new subpage on https://github.com/hpcugent/hpcugent.github.com ?

**Comment by boegel** [comment]
@JensTimmerman: because we're not the only ones organising this devroom, it should be "on neutral ground", i.e. http://hpc-bigdata-fosdem16.github.io for example.

Also, feel free to ignore PRs in this repo, @ehiggs and I will handle this together with @rvs and @vasia. :-)

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+### content configuration ###
+title:       "FOSDEM'16 HPC, Big Data, and Data Science Devroom"
+keywords:    "hpc, big data, data science"
+description: "FOSDEM'16 HPC, Big Data, and Data Science Devroom"
+source_link: "https://github.com/hpcugent/fosdem2016_hpc_bigdata_devroom"
+#favicon:     "img/favicon.ico"
+#touch_icon:  "img/apple-touch-icon.png"
+google_analytics_key: UA-55665759-3
```
Comment (ehiggs) [inline _config.yml]: Derp, this isn't my key. I don't use GA. Does anyone want stats or shall I remove this?

Comment (vasia) [inline _config.yml]: I'm fine with removing it.

File: _posts/2015-01-01-intro.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# FOSDEM&#39;16 HPC, Big Data, and Data Science Devroom
+
+<div style="text-align:center;">
+  <span class="fa-stack subtlecircle" style="font-size:64px; background:rgba(0,128,0,0.1)">
+    <i class="fa fa-circle fa-stack-2x text-white"></i>
+    <i class="fa fa-server fa-stack-1x text-green"></i>
+  </span>
+</div>
+
+### January 30, 2016, Brussels, Belgium.
```
Comment (vasia) [inline _posts/2015-01-01-intro.md]: this should be Jan 31

File: _posts/2015-01-03-submissions.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Submission"
+bg: orange
+color: white
+style: left
+fa-icon: file-text-o
+icon-title: true
+---
+
+We invite presenters to submit original, high-quality work with
```
Comment (vasia) [inline _posts/2015-01-03-submissions.md]: I would actually skip the word "original". This is a devroom, not an academic conference and I guess we can't expect people to only propose original talks?

File: _posts/2015-01-05-committees.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+## Organizing Committee
+
+* [Ewan Higgs]()
+* [Kenneth Hoste]()
+* [Roman Shaposhnik]()
+* [Vasia Kalavri]()
+
+For the CoC we recommend using the [Apache Code of Conduct](
```
Comment (vasia) [inline _posts/2015-01-05-committees.md]: I don't think "recommend" sends the right message here. It seems as if we're saying "we recommend you behave, but we won't enforce it". How about a simple "We adopt/use the Apache Code of Conduct"?

**Comment by ehiggs** [comment]
@boegel, I went ahead and made the Organization you suggested and uploaded the stuff there.

**Comment by boegel** [comment]
@ehiggs: so, this repo is useless? better trash it then, to avoid confusion?

**Comment by vasia** [comment]
Thanks @ehiggs! Where can I see the website now? https://hpc-bigdata-fosdem16.github.io doesn't seem to work..

**Comment by ehiggs** [comment]
Work is here: https://github.com/hpcugent/hpcugent.github.com

**Pull request closed without merging**